android/build.gradle

```
@@ -9,7 +9,7 @@ buildscript {
         maven { url '"https://oss.sonatype.org/content/repositories/snapshots' }
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:$versions.gradle"
+        classpath "com.android.tools.build:gradle:3.4.2"
```

```suggestion
        classpath "com.android.tools.build:gradle:$versions.gradle"
```
`$versions.gradle` devrait être utilisée.